data/maps/SP_Scenarios/Along The River.wmf/scripting/objectives.lua

```diff
@@ -0,0 +1,84 @@
+-- =======================================================================
+--                             Along The River
+-- =======================================================================
+
+obj_defeat_red = {
+   name = "defeat_red",
+   title = _"Defeat Törver",
+   number = 1,
+   body = objective_text(_"Defeat the armies of Törver.",
+      li_image("map:genstats_enable_plr_02.png", _[[]])
```

Comment:
Never translate an empty string! Several more below.

Reply:
It was meant as a placeholder. Will ad some text here, explaining the red flag shown. Like "He is the enemy in the north, fighting under the red banner."